stdlib/tarfile.pyi

```diff
@@ -260,7 +259,24 @@ def open(
     pax_headers: Mapping[str, str] | None = ...,
     debug: int | None = ...,
     errorlevel: int | None = ...,
-    preset: int | None = ...,
+) -> TarFile: ...
```

Comment:
I don't get why modes `w|` or `w|xz` couldn't pass `compresslevel`, it just goes unused doesn't it ? Or are we being so restrictive with this function as to not allow unused parameters with certain modes?
Source: https://github.com/python/cpython/blob/main/Lib/tarfile.py#L1889

Reply:
Yes, it's intended to prevent a misunderstanding of how this works (I.e. a subtle if harmless bug).